geode-lucene/src/main/java/org/apache/geode/cache/lucene/internal/LuceneEventListener.java

```diff
@@ -34,7 +34,6 @@
 import org.apache.geode.cache.query.internal.DefaultQuery;
 import org.apache.geode.internal.cache.BucketNotFoundException;
 import org.apache.geode.internal.cache.PrimaryBucketException;
-import org.apache.geode.internal.cache.tier.sockets.command.Default;
```

Comment:
I reverted the first set of changes to this file as it didn't have anything to do with this change set.  It shouldn't be showing up in this diff anymore but if it is I'll make sure to revert it...